### PR TITLE
Created Backup message to run when NASA API is down

### PR DESCRIPTION
**Pull request by Daniel-olaO**

### CODE REVIEW

File: api/views.py [hunk 1]
```diff
@@ -195,3 +190,57 @@ def sendWelcomeMessage(name, number):
...
+                media_url=data['url']
+            )
+            print(message.sid)
+        else:
+            message = client.messages.create(
+            to=number,
+            from_=os.environ.get('TWILIO_PHONE_NUMBER'),
+            body= f'\nToday\'s NASA Astronomy Picture of the Day is: {data["title"]}.\n\n{data["explanation"]} \n\n Video URL: {data["url"]}'
+            )
+            print(message.sid)
```
Comment (Russell-Pollari) [inline api/views.py]: you are duplicating the logic to send an apod text here. Maybe instead do something like:

```python
def send_apod_text(apod_data: dict, number: str):
    # Connect to Twilio and create text from apod data
```

And then you can do something like this when sending the daily text 

```python
data = get_nasa_apod()
if data is None:
   data = get_random_apod()

send_apod_text(data)
```

Note I'm using the python naming convention of `snake_case` in my examples

File: api/scheduler/scheduler.py [hunk 2]
```diff
@@ -1,11 +1,14 @@
 import datetime
 from apscheduler.schedulers.background import BackgroundScheduler
-from ..views import sendAPOD
+from apscheduler.triggers.cron import CronTrigger
+from ..views import sendAPOD, storeMessageWeekly
```
Comment (Russell-Pollari) [inline api/scheduler/scheduler.py]: fine for now, but you should probably put these in a different file, `views` in a Django rest project is expected to contain API routes

**Review by Russell-Pollari — CHANGES_REQUESTED** [review]

File: api/views.py (hunk 1 above)
Comment (Daniel-olaO) [reply api/views.py]: I can use snake_case for all of my functions

File: api/scheduler/scheduler.py (hunk 2 above)
Comment (Daniel-olaO) [reply api/scheduler/scheduler.py]: I followed the instructions from the docs

Comment (Russell-Pollari) [reply api/scheduler/scheduler.py]: which instructions?

File: api/views.py [hunk 3]
```diff
@@ -146,61 +146,70 @@ def nasaAPOD():
     except requests.exceptions.RequestException as requestException:
         print("RequestException: ", requestException)
 
-def sendText(number):
+def send_apod_text(apod_data: dict, phone_number: str):
```
Comment (Russell-Pollari) [inline api/views.py:154]: nice type annotations!

**Review by Russell-Pollari — APPROVED** [review]

**Pull request merged**